crds/crd-coralogix-alert-set.yaml

```diff
@@ -288,12 +286,23 @@ spec:
                                     end:
                                       type: string
                                       x-scala-type: Time
-
                       notificationPayloadFilters:
                         type: array
                         items:
                           type: string
                           x-scala-type: PayloadFilter
+                      labels:
```

Comment:
Can you put it in the start of the CRD? like in the UI order, it should come after severity